### PR TITLE
ClasspathHelper.forClassLoader(...) shouldn't reorder URLs

### PR DESCRIPTION
**Pull request by g7r**

### CODE REVIEW

**Comment by g7r** [comment]
Hello @ronmamo !
The issue I'm fixing is effectively blocking our usage of vanilla Reflections in our projects since 2013. Looking forward for my pull request to be accepted.

File: src/main/java/org/reflections/util/ClasspathHelper.java [hunk 1]
```diff
@@ -12,12 +12,7 @@
 import java.net.URL;
 import java.net.URLClassLoader;
 import java.net.URLDecoder;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Enumeration;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (martin-g) [inline src/main/java/org/reflections/util/ClasspathHelper.java]: Please revert that change. Import all classes explicitly.

**Comment by martin-g** [comment]
The PR looks good to me.

**Comment by ronmamo** [comment]
gr8 @g7r!

**Pull request merged**